modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -145,7 +145,7 @@ namespace {{packageName}}.Client
         /// <param name="contentType">Content Type of the request</param>
         /// <returns>Object</returns>
         public Object CallApi(
-            String path, RestSharp.Method method, Dictionary<String, String> queryParams, Object postBody,
+            String path, RestSharp.Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
```

Comment:
@jimschubert definitely valid points. I've heard less than 3 use cases in which the users need to extend the API clients to override the `CallApi` and `CallApiAsync` method but still we should avoid breaking their usage if possible.